packages/server/integrations/jiraServer/JiraServerRestManager.ts

```diff
@@ -54,45 +90,104 @@ export default class JiraServerRestManager {
     return json.errorMessages?.join('\n')
   }
 
-  async request(method: string, path: string) {
+  async parseJsonResponse<T>(response: Response): Promise<T | Error> {
+    const contentType = response.headers.get('content-type') || ''
+    if (!contentType.includes('application/json')) {
+      return new Error('Received non-JSON Jira Server Response')
+    }
+    const json = await response.json()
+
+    if (response.status !== 201 && response.status !== 200) {
+      console.error(`Jira server error ${response.status}`, json)
```

Comment:
+1 do we need this additional `console.error`?

Reply:
Removed